Dockerfile

```diff
@@ -116,6 +116,16 @@ RUN echo ". /sai-challenger/scripts/sai-cli-completion.sh" >> /root/.bashrc
 
 RUN pip3 install pytest pytest_dependency pytest-html
 
+
+# Deploy SAI Challenger CLI
+ADD . /sai-challenger/
+RUN ln -s /sai-challenger /usr/local/lib/python3.7/dist-packages/saichallenger
+RUN echo ". /sai-challenger/scripts/sai-cli-completion.sh" >> /root/.bashrc
+
+# Install SAI Challenger CLI and the remote commands listener
+RUN pip3 install --editable /sai-challenger/common /sai-challenger
```

Comment:
For a standalone mode this is done through supervisord.conf
https://github.com/opencomputeproject/SAI-Challenger/blob/beece67f68142de5b7486f5a839eacd7010fe2b2/configs/supervisord.conf#L25